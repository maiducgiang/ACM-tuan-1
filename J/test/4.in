2
74,23(96)
73,39(27)
