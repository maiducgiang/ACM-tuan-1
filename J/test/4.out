244991 3300
20183 275
